functions/src/config.ts

```diff
@@ -156,6 +156,7 @@ export interface SendVoteToManagerConfig {
 
 export interface DailyAlibeezImportConfig {
   enabled: Flag;
+  batch_size: number;
```

Comment:
Should be string because firebase config only supports strings. CI is red because of that. See https://circleci.com/gh/Zenika/humeur-du-mois-2018/3392?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link

Reply:
woops, will fix